plugins/sagemaker/scripts/docker_build.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
```

Comment:
Do we need this? can't we use flytekit_build_image.sh? cc @wild-endeavor

Reply:
interesting, if we can, then we should just replace it in all the places

Reply:
lets do this as a separate pass?